src/main/java/TaskList.java

```diff
@@ -0,0 +1,120 @@
+import java.util.ArrayList;
+
+/**
+ * Represents the list of tasks in the program.
+ */
+public class TaskList {
```

Comment:
Considering the way you're adding get, add and remove methods, you might want to consider having TaskList extend ArrayList<Task> like what I did